src/parseFunctions.js

```diff
@@ -16,4 +16,32 @@ const parseQueryToJson = (queryString) => {
   return obj;
 };
 
-export default parseQueryToJson;
+const parseHeaderToJSON = (headerString) => {
+  const pairs = headerString.split('\n');
+  const result = {};
+  pairs.forEach((pair) => {
+    const [key, value] = pair.split(':');
+    if (CMCDheaders.includes(key)) {
+      if (value !== undefined) {
+        const subPairs = value.split(',');
+        subPairs.forEach((subPair) => {
+          subPair = subPair.replace(/ /g, '');
+          if (!subPair.includes('=')) {
+            result[subPair] = true;
+          } else {
+            let [subKey, subValue] = subPair.split('=');
+            if (subValue !== undefined) {
```

Comment:
check if changing to this is better =>` if (value)`

Reply:
I just checked cases if we get a falsy value and there was no problem so I'm going to do ir, thanks for your contribution!